source/ed/data.h

```diff
@@ -99,6 +99,8 @@ class Data: boost::noncopyable {
 
     std::vector<ed::types::AdminStopArea*>  admin_stop_areas;
 
+    std::vector<ed::types::ObjectProperty*> object_properties;
```

Comment:
* for the `vector` recopy when using `push_back`, most of the time, it's not a problem. For cases when it's a problem, using `std::deque` is still better than using `unique_ptr` (because of the number of allocation). using a `std::vector<unique_ptr>` can be useful when you use pointer to these objects elsewhere as a `push_back` may invalidate the pointer.
* Most of the collections are deleted in the destructor of `ed::Data::~Data` (with the magical macro). We just saw that `stop_point_connections` and `admin_stop_areas` are leaked, and that's a bug.
* We prefer your 2) propositions.
* The existing leaks are a bug, and we must fix them (that's not your responsibility)
* using `std::vector<unique_ptr>` for the `ED_COLLECTIONS` would be a great cleanup of the code, but not a bug correction as there is no leak.